main.go

```diff
@@ -55,9 +53,6 @@ var (
 
 func init() {
 	flag.Parse()
-	if *vaultToken == "" {
```

Comment:
Since this check is removed, perhaps we should still assert vault token not empty in the actual vault module when vault is accessed? I think all four read/write methods in this file https://github.com/cruise-automation/isopod/blob/master/pkg/vault/vault.go#L63 should have some check similar to 
```go
if p.client.Token() == "" {
    return errors.New("Tried to access Vault but token is empty. Please set --vault_token or $VAULT_TOKEN")
} 
```
and maybe extract this to a short function.